### PR TITLE
Added Local Variable and Global Variable scripts

### PR DESCRIPTION
**Pull request by saysaad**

### CODE REVIEW

File: sayeed/localvariable.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+#Author:Ahmed Sayeed
+
+getName(){
+local NAME="Ahmed"
+local apple="fruit"
+
+echo "$NAME (from function) " #valid if called using function
+
```
Comment (mateen649) [inline sayeed/localvariable.sh]: Remove this extra line

File: sayeed/localvariable.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+#Author:Ahmed Sayeed
+
+getName(){
+local NAME="Ahmed"
+local apple="fruit"
+
```
Comment (mateen649) [inline sayeed/localvariable.sh]: Remove this extra line

File: sayeed/localvariable.sh (hunk 1 above)
Comment (saysaad) [reply sayeed/localvariable.sh]: Removed

File: sayeed/localvariable.sh (hunk 2 above)
Comment (saysaad) [reply sayeed/localvariable.sh]: resolved

**Comment by mateen649** [comment]
Looks good. PR Approved

**Pull request merged**